crates/sui-protocol-config/src/lib.rs

```diff
@@ -522,6 +523,10 @@ pub struct ProtocolConfig {
     hmac_hmac_sha3_256_cost_base: Option<u64>,
     hmac_hmac_sha3_256_input_cost_per_byte: Option<u64>,
     hmac_hmac_sha3_256_input_cost_per_block: Option<u64>,
+
+    // Config introduced in V2
```

Comment:
This doesn't seem like a very useful comment. Maybe have a descriptive comment about what this means, and put it next to the other config params that are related?